siriuspy/siriuspy/devices/device.py

```diff
@@ -200,6 +200,7 @@ def comp_(val):
         if comp_(value):
             return True
 
+        timeout = timeout if timeout else _DEF_TIMEOUT
```

Comment:
Why do we need this line, @xresende ? The default value of `timeout` already is `_DEF_TIMEOUT`